components/Breadcrumb/src/breadcrumb.css

```diff
@@ -0,0 +1,121 @@
+.denhaag-breadcrumb {
+  --denhaag-breadcrumb-background-color: transparent;
+  color: var(--denhaag-breadcrumb-color, #4b4b4b);
```

Comment:
In a few places there are still hardcoded colors. There are design choices and should in my opinion be discarded. It's OK to expect other organisations to define certain tokens.

Reply:
Replaced by inherit over the whole document. Since otherwise we overwrite a value with an optional one, which can cause issues.